ci-operator/config/fabric8-services/toolchain-operator/fabric8-services-toolchain-operator-master.yaml

```diff
@@ -14,7 +14,10 @@ resources:
     requests:
       memory: 250Mi
       cpu: 100m
-
+promotion:
+  cluster: https://api.ci.openshift.org
+  namespace: openshift
```

Comment:
You need to promote to a different namespace.

Reply:
@smarterclayton thanks for your comments. addressed it in 261ffa1